### DISCUSSION TITLE
Add RPZ policy name

### DISCUSSION THREAD

**Discussion by dmgeurts** [dmachard/DNS-collector#130] [Ideas]
PowerDNS-recursor will parse the policy if a request was matched (and likely subsequently blocked or altered).

Logging these policy maps is important when tracking which DNS clients made requests to blocked domains, and there are other uses of RPZ feeds where logging like this is equally useful.

[Finally, if a RPZ or custom Lua policy has been applied, response messages also contain the applied policy name and some tags. This is particularly useful to detect and act on infected hosts.](https://doc.powerdns.com/recursor/lua-config/protobuf.html)

Please add support for the logging of policy names.

**Comment by dmachard** [comment]
I just tested with the following configuration on the PowerDNS recursor

```
rpzFile("/etc/pdns-recursor/basic.rpz", {
  policyName="custom",
  tags={"tagA", "tagB"}
})

```

This enhancement will be available with the next release.

**Comment by dmachard** [comment]
new pull request in progress https://github.com/dmachard/go-dns-collector/pull/131

**Reply by dmachard** [reply]
New beta release [0.24.0-beta1](https://github.com/dmachard/go-dns-collector/releases/tag/v0.24.0-beta1 ) available 
with PowerDNS enhancement regarding the protobuf support.

This release implement the new following attributes to log policy and tag(s):
- `pdns-tag` and `pdns-tags`
- `pdns-applied-policy`

Any feedbacks will be appreciated.

**Comment by dmachard** [comment]
resolved in release 0.24.0
